### PR TITLE
feature/1629 - Rework calendar as a wrapper around p-calendar that overwrites the control with a new control that updates on submit and has custom validation procedures for whenever a value changes

### PR DESCRIPTION
**Pull request by sasha-dresden**
Issue [FECFILE-1629](https://fecgov.atlassian.net/browse/FECFILE-1629)

### CODE REVIEW

File: front-end/cypress/e2e/models/ContactFormModel.ts
Comment (Elaine-Krauss-TCG) [inline front-end/cypress/e2e/models/ContactFormModel.ts:1]: Great change 👍

File: front-end/src/app/shared/utils/subscription-form-control.ts
Comment (Elaine-Krauss-TCG) [inline front-end/src/app/shared/utils/subscription-form-control.ts:1]: Absolutely brilliant addition 🎉

**Review by Elaine-Krauss-TCG — APPROVED** [review]
Code looks good, no issues were raised in my smoke testing, and all E2E tests pass locally

**Comment by sasha-dresden** [comment]
Closing PR as I created a new [PR2370](https://github.com/fecgov/fecfile-web-app/pull/2370) where this is being handled by a new [ticket](https://fecgov.atlassian.net/browse/FECFILE-1786) as [1629](https://fecgov.atlassian.net/browse/FECFILE-1629) was passed as it was.

**Pull request closed without merging**